packages/core/src/test/codewhisperer/commands/transformByQ.test.ts

```diff
@@ -67,12 +65,6 @@ describe('transformByQ', function () {
     })
 
     it('WHEN job is stopped THEN status is updated to cancelled', async function () {
-        const testWindow = getTestWindow()
-        testWindow.onDidShowMessage(message => {
-            if (message.message === stopTransformByQMessage) {
-                message.selectItem(startTransformByQ.stopTransformByQButton)
```

Comment:
Deleted since we no longer have this pop-up asking users to confirm to stop their job.